lib/live_view_native/platforms.ex

```diff
@@ -20,7 +20,7 @@ defmodule LiveViewNative.Platforms do
   Provides configuration constants about all platforms supported by an
   application that uses LiveView Native. This function is a dependency
   of various LiveView Native systems, such as `LiveViewNative.LiveSession`
-  which is responsible for determining which platform (web, iOS, etc.) a
+  which is responsible for determining which platform (HTML, iOS, etc.) a
```

Comment:
Can you change `iOS` to `SwiftUI`

Reply:
Will do.

note besides which Im sure y'all already know that updating `platform_id: :html` being a [breaking change to existing / previous repos](https://github.com/search?q=org%3Aliveview-native+%3Aweb&type=code&p=3), there are still a good amount of references to web throughout the codebase like [# This UI renders on the web](https://github.com/liveview-native/live_view_native/blob/d0c68ccf51773c27c60b50daa0d173b13407ea42/guides/introduction/your-first-native-liveview.md?plain=1#L26) and [Hello web!](https://github.com/liveview-native/live_view_native/blob/d0c68ccf51773c27c60b50daa0d173b13407ea42/guides/introduction/your-first-native-liveview.md?plain=1#L26), etc that Im not sure if needs updating...

Reply:
@cohawk This being a breaking change shouldn't be a concern since the `next` branch will already includes breaking changes and the `:web` / `:html` platform is more of an internal thing than something LVN apps depend on. All of the documentation and examples present the web `render/1` function as the fallback case rather than explicitly matching on `:platform_id` so I don't expect apps in the wild to be relying on this.